### PR TITLE
History management, user profile

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Added videos to the history page
Remove Video From History Page
Remove All Videos from the history page
Added User Profile Page

image
image
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -28,33 +28,39 @@ body {
...
   --secondary-color: #571089;
+  --navbar-primary-color:#272b33;
+  --trash-color:#ff0000;
+  --count-desc:#58a6ff;
 }
 
 .dark__mode {
-  --primary-color: #141319;
+  --primary-color: #0a0c10;
+  /* --navbar-primary-color:#272b33; */
```
Comment (samyakshah3008) [inline src/App.css:47]: Can remove this commented code

File: src/App.css [hunk 2]
```diff
@@ -28,33 +28,39 @@ body {
...
   --bg-code: #fafafa;
   --black-bg: #000814;
   --borders-color: rgb(149, 54, 238);
   --footer-color: #14213d;
   --border-color: rgb(149, 54, 238);
+  --count-desc:#58a6ff;
 }
 
 .light__mode {
+  /* --navbar-primary-color:#fafafa; */
```
Comment (samyakshah3008) [inline src/App.css:57]: Can remove this

**Review by samyakshah3008 — APPROVED** [review]
Really liked your UI, color contrast is ✅ 
Nice and clean code, but I see few unncessary comments, you can remove later while refactoring
Good use of axios calling in service folder, that made your code pretty clean 
Good naming conventions 

Good for approval

**Review by jeetbhalani15 — APPROVED** [review]
Context For History Works Fine.
- Code looks clean
- Naming & Folders looks good

File: src/App.css (hunk 1 above)
Comment (ranitasahaa1230) [reply src/App.css:47]: Sure will remove all these lines

**Pull request merged**